code/modules/mining/abandoned_crates.dm

```diff
@@ -149,6 +149,234 @@
+		if(127)
+			new /obj/item/clothing/head/crown/fancy(src)
+			new /obj/item/storage/box/monkeycubes(src)
+			new /obj/item/storage/box/monkeycubes(src)
+			new /obj/item/storage/box/monkeycubes(src)
+		if(128)
+			switch(pickweight(list("midstuff1" = 1, "midstuff2" = 1, "midstuff3" = 1, "midstuff4" = 1, "midstuff5" = 2, "midstuff6" = 1, "midstuff7" = 2, "midstuff8" = 1, "midstuff9" = 1)))
+				if("midstuff1")
+					new /obj/item/storage/firstaid/tactical(src)
+					new /obj/item/shield/riot/tele(src)
+					new /obj/item/pickaxe/drill/diamonddrill(src)
+				if("midstuff2")
+					new /obj/item/shield/energy(src)
+				if("midstuff3")
+					new /obj/item/dnainjector/lasereyesmut(src)
+				if("midstuff4")
+					new /obj/item/storage/backpack/holding(src)
+					new /obj/item/grenade/spawnergrenade/spesscarp(src)
+					new /obj/item/grenade/spawnergrenade/manhacks(src)
+				if("midstuff5")
+					new /obj/item/stack/sheet/mineral/diamond{amount = 15}(src)
+					new /obj/item/stack/sheet/mineral/uranium{amount = 15}(src)
+					new /obj/item/stack/sheet/mineral/plasma{amount = 15}(src)
+					new /obj/item/stack/sheet/mineral/gold{amount = 15}(src)
+				if("midstuff6")
+					new /obj/item/book/granter/spell/barnyard(src)
+				if("midstuff7")
+					new /obj/item/borg/upgrade/vtec(src)
+					new /obj/item/borg/upgrade/disablercooler(src)
+				if("midstuff8")
+					new /obj/item/book/granter/spell/knock(src)
+				if("midstuff9")
+					new /obj/item/book/granter/spell/summonitem(src)
+					new /obj/item/book/granter/spell/forcewall(src)
+		if(129)
+			new /obj/item/stack/sheet/mineral/snow{amount = 25}(src)
+			new /obj/item/toy/snowball(src)
+			new /obj/item/twohanded/spear(src)
+		if(130)
+			new /mob/living/simple_animal/bot/medbot(src)
+			new /mob/living/simple_animal/bot/floorbot(src)
+			new /mob/living/simple_animal/bot/cleanbot(src)
+		if(131)
+			new /obj/item/card/emagfake(src)
+			new /obj/item/ammo_box/magazine/toy/pistol/riot(src)
+			new /obj/item/gun/ballistic/automatic/toy/pistol/riot(src)
+			new /obj/item/toy/sword(src)
+			new /obj/item/toy/cards/deck/syndicate(src)
+			new /obj/item/toy/syndicateballoon(src)
+			new /obj/item/bedsheet/syndie(src)
+			new /obj/item/toy/plush/nukeplushie(src)
+			new /obj/item/toy/nuke(src)
+			new /obj/item/disk/nuclear/fake(src)
+		if(132)
+			new /obj/item/pneumatic_cannon/speargun(src)
+		if(133)
+			new /obj/item/dnainjector/elvismut(src)
+			new /obj/item/dnainjector/firemut(src)
+			new /obj/item/dnainjector/glow(src)
+			new /obj/item/dnainjector/spastic(src)
+			new /obj/item/dnainjector/epimut(src)
+			new /obj/item/handmirror(src)
+			new /obj/item/razor(src)
+		if(134 to 138)
+			for(var/stick in 7)
+				stick = pick(subtypesof(/obj/item/flashlight/glowstick))
+				new stick(src)
+		if(139 to 141)
+			new /obj/effect/spawner/lootdrop/two_percent_xeno_egg_spawner(src)//Two percent for a crate to spawn a two percent xeno spawner.
```

Comment:
Horrible idea, Xenomorphs are guaranteed round enders most of the time.

Reply:
I'm not memorizing the multiplication table, but I can say the chance is even lower than station's xenobio egg spawner.